### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by hvolschenk**
* linting using eslint
* testing using jest
* add docker environment
* initial get command

### CODE REVIEW

File: .azure/build-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+stages:
+
+- stage: Build
+  pool:
+    vmImage: ubuntu-16.04
+  variables:
+  - group: node-cli--build
+  jobs:
+  - job: BuildBFF
+    displayName: Build and push BFF
```
Comment (hvolschenk) [inline .azure/build-pipeline.yml]: Remove _BFF_ from these.

File: mundophrase/en-US.json [hunk 2]
```diff
@@ -0,0 +1,283 @@
+{
```
Comment (hvolschenk) [inline mundophrase/en-US.json]: Remove this file.

**Review by helloserve — APPROVED** [review]

File: .azure/build-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  variables:
+  - group: node-cli--build
+  jobs:
+  - job: Build
+    displayName: Test and publish
+    steps:
+    - script: |
+        docker build \
+          --tag ci-runner \
+          client
```
Comment (hvolschenk) [inline .azure/build-pipeline.yml]:
```suggestion
          .
```

File: .azure/build-pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    vmImage: ubuntu-16.04
+  variables:
+  - group: node-cli--build
+  jobs:
+  - job: Build
+    displayName: Test and publish
+    steps:
+    - script: |
+        docker build \
+          --tag ci-runner \
```
Comment (hvolschenk) [inline .azure/build-pipeline.yml]:
```suggestion
          --tag build-runner \
```

File: .azure/build-pipeline.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+          client
+      displayName: Prepare CI Docker image
+    - script: |
+        docker run \
+          --rm \
+          --user "$(id -u):$(id -g)" \
+          --volume "$(pwd):/home/node/app" \
+          ci-runner \
```
Comment (hvolschenk) [inline .azure/build-pipeline.yml]:
```suggestion
          build-runner \
```

File: .azure/build-pipeline.yml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+          npm install
+      displayName: Install dependencies
+    - script: |
+        docker run \
+          --rm \
+          --user "$(id -u):$(id -g)" \
+          --volume "$(pwd):/home/node/app" \
+          ci-runner \
```
Comment (hvolschenk) [inline .azure/build-pipeline.yml]:
```suggestion
          build-runner \
```

File: .azure/build-pipeline.yml [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+          npm test
+      displayName: Run tests
+    - script: |
+        docker run \
+          -e "GITHUB_TOKEN=${GITHUB_TOKEN}" \
+          -e "NPM_TOKEN=${NPM_TOKEN}" \
+          --rm \
+          --user "$(id -u):$(id -g)" \
+          --volume "$(pwd):/home/node/app" \
+          ci-runner \
```
Comment (hvolschenk) [inline .azure/build-pipeline.yml]:
```suggestion
          build-runner \
```

**Pull request merged**